src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -297,9 +297,8 @@ func NonStopWriter(writers ...io.Writer) io.Writer {
 // It never fails and always return the length of bytes passed in
 func (t *nonStopWriter) Write(p []byte) (int, error) {
 	for _, w := range t.writers {
-		if n, err := w.Write(p); err != nil {
-			return n, err
-		}
+		// not checking for errors
+		w.Write(p)
```

Comment:
Yes that was the reason. I introduced this in a specific "fix", which I don't understand in retrospect. But certainly makes no sense at the moment.
I did all changes in that PR trying to leave all behavior untouched, otherwise.